sandbox/pages/SandboxCodeBlock.vue

```diff
@@ -176,18 +203,35 @@
 </template>
 
 <script setup lang="ts">
-import { inject } from 'vue'
+import { computed, ref, inject, onBeforeMount, watch, nextTick } from 'vue'
+import type { CodeBlockEventData } from '@/types'
+import useShiki from '../composables/useShiki'
 import SandboxTitleComponent from '../components/SandboxTitleComponent.vue'
 import SandboxSectionComponent from '../components/SandboxSectionComponent.vue'
 
-const code = `{
+const { createHighlighter, highlighter } = useShiki()
+
+const highlightedToggle = ref(true)
+const origLines = [1,2,3]
```

Comment:
super-nit: can you start the highlighting on a line besides 1 since that's just an opening bracket